t/03-testapi.t

```diff
@@ -624,6 +631,27 @@ subtest 'script_sudo' => sub {
             }
     ]);
     $cmds = [];
+    script_sudo("ls /", 0);
+    is_deeply($cmds, [
+            {
+                text => "sudo ls /\n",
+                cmd => 'backend_type_string'
+            },
+            {
+                mustmatch => 'sudo-passwordprompt',
+                cmd => 'check_screen',
+                timeout => 3,
+                check => 1,
+                no_wait => undef
+            },
+            {
+                cmd => 'backend_type_string',
+                secret => 1,
+                text => 'stupid',
+                max_interval => 100
+            }
+    ]);
```

Comment:
I wouldn't care to check the complete output in detail. It's enough to just check some specific parts, e.g.

```suggestion
    is $cmds[0]{text}, 'sudo ls /\n',
    is_deeply $cmds[0], {text => 'sudo ls /\n', cmd => 'backend_type_string'}, 'sudo command is typed';
    ok $cmds[2]{secret}, 'password is treated as secret';
```

Reply:
you did not update the code. Maybe forgot to commit your local changes?